
# TOC


- mol2text
- Docking
- other tools


# molecule 2 text

## Papers
1. [2021 EMNLP] Text2Mol: Cross-Modal Molecule Retrieval with Natural Language Queries [[Paper]](https://aclanthology.org/2021.emnlp-main.47/) [[Code]](https://github.com/cnedwards/text2mol)
2. [2022 EMNLP] Translation between Molecules and Natural Language (MolT5) [[Paper]](https://aclanthology.org/2022.emnlp-main.26/) [[Code]](https://github.com/blender-nlp/MolT5)
3. [2022 Nature Communications] A deep-learning system bridging molecule structure and biomedical text with comprehension comparable to human professionals [[Paper]](https://www.nature.com/articles/s41467-022-28494-3) [[Code]](https://github.com/thunlp/KV-PLM)
4. [2022 Nature Machine Intelligence] MoleculeSTM: Multi-modal Molecule Structure-text Model for Text-based Retrieval and Editing [[Paper]](https://arxiv.org/abs/2212.10789) [[Code]](https://github.com/chao1224/MoleculeSTM)
5. [2022 arXiv] A Molecular Multimodal Foundation Model Associating Molecule Graphs with Natural Language [[Paper]](https://arxiv.org/abs/2209.05481) [[Code]](https://github.com/BingSu12/MoMu)
6. [2023 ICML] Unifying Molecular and Textual Representations via Multi-task Language Modeling [[Paper]](https://arxiv.org/abs/2301.12586) [[Code]](https://github.com/GT4SD/multitask_text_and_chemistry_t5) 
7. [2023 ICML] Enhancing Activity Prediction Models in Drug Discovery with the Ability to Understand Human Language [[Paper]](https://proceedings.mlr.press/v202/seidl23a/seidl23a.pdf) [[Code]](https://github.com/ml-jku/clamp)
8. [2023 WCB @ ICML] Extracting Molecular Properties from Natural Language with Multimodal Contrastive Learning [[Paper]](https://arxiv.org/abs/2307.12996)
9. [2023 NeurIPS] GIMLET: A Unified Graph-Text Model for Instruction-Based Molecule Zero-Shot Learning [[Paper]](https://arxiv.org/abs/2306.13089) [[Code]](https://github.com/zhao-ht/GIMLET)
10. [2023 NeurIPS Benchmark & Dataset] What can Large Language Models do in chemistry? A comprehensive benchmark on eight tasks [[Paper]](https://arxiv.org/abs/2305.18365) [[Code]](https://github.com/ChemFoundationModels/ChemLLMBench)
11. [2023 ACL Short] MolXPT: Wrapping Molecules with Text for Generative Pre-training [[Paper]](https://aclanthology.org/2023.acl-short.138/)
12. [2023 EMNLP] MolCA: Molecular Graph-Language Modeling with Cross-Modal Projector and Uni-Modal Adapter [[Paper]](https://arxiv.org/abs/2310.12798) [[Code]](https://github.com/acharkq/MolCA)
13. [2023 EMNLP] BioT5: Enriching Cross-modal Integration in Biology with Chemical Knowledge and Natural Language Associations [[Paper]](https://arxiv.org/abs/2310.07276) [[Code]](https://github.com/QizhiPei/BioT5)
14. [2023 EMNLP] Predictive Chemistry Augmented with Text Retrieval [[Paper]](https://arxiv.org/abs/2312.04881) [[Code]](https://github.com/thomas0809/textreact)
15. [2023 EMNLP Findings] ReLM: Leveraging Language Models for Enhanced Chemical Reaction Prediction [[Paper]](https://arxiv.org/pdf/2310.13590.pdf) [[Code]](https://github.com/syr-cn/relm)
16. [2023 arXiv] MolFM: A Multimodal Molecular Foundation Model [[Paper]](https://arxiv.org/abs/2307.09484) [[Code]](https://github.com/PharMolix/OpenBioMed)
17. [2023 arXiv] Can Large Language Models Empower Molecular Property Prediction? [[Paper]](https://arxiv.org/abs/2307.07443) [[Code]](https://github.com/chnq/llm4mol)
18. [2023 arXiv] Interactive Molecular Discovery with Natural Language [[Paper]](https://arxiv.org/abs/2306.11976) [[Code]](https://github.com/Ellenzzn/ChatMol/tree/main)
19. [2023 arXiv] Empowering Molecule Discovery for Molecule-Caption Translation with Large Language Models: A ChatGPT Perspective [[Paper]](https://arxiv.org/abs/2306.06615) [[Code]](https://github.com/phenixace/MolReGPT)
20. [2023 arXiv] From Artificially Real to Real: Leveraging Pseudo Data from Large Language Models for Low-Resource Molecule Discovery [[Paper]](https://arxiv.org/abs/2309.05203)
20. [2023 arXiv] GIT-Mol: A Multi-modal Large Language Model for Molecular Science with Graph, Image, and Text [[Paper]](https://arxiv.org/abs/2308.06911)
21. [2023 arXiv] Catalyst Property Prediction with CatBERTa: Unveiling Feature Exploration Strategies through Large Language Models [[Paper]](https://arxiv.org/abs/2309.00563) [[Code]](https://github.com/hoon-ock/CatBERTa)
22. [2023 arXiv] GPT-MolBERTa: GPT Molecular Features Language Model for molecular property prediction [[Paper]](https://arxiv.org/abs/2310.03030) [[Code]](https://github.com/Suryanarayanan-Balaji/GPT-MolBERTa)
23. [2024 ICLR] Mol-Instructions: A Large-Scale Biomolecular Instruction Dataset for Large Language Models [[Paper]](https://arxiv.org/abs/2306.08018) [[Code]](https://github.com/zjunlp/Mol-Instructions)
24. [2024 ICLR] Towards 3D Molecule-Text Interpretation in Language Models [[Paper]](https://openreview.net/forum?id=xI4yNlkaqh) [[Code]](https://github.com/lsh0520/3D-MoLM)
25. [2024 ICLR] Conversational Drug Editing Using Retrieval and Domain Feedback [[Paper]](https://openreview.net/forum?id=yRrPfKyJQ2) [[Code]](https://github.com/chao1224/ChatDrug)
26. [2024 ICLR rejected] Text2Data: Low-Resource Data Generation with Textual Control [[Paper]](https://openreview.net/forum?id=Y2Txh5uGRe)
27. [2024 ICLR rejected] Text-guided Diffusion Model for 3D Molecule Generation [[Paper]](https://openreview.net/forum?id=FdUloEgBSE)
28. [2024 ICLR rejected] Form follows Function: Text-to-Text Conditional Graph Generation based on Functional Requirements [[Paper]](https://openreview.net/forum?id=Pu3qMB9aKD)
29. [2024 ICML] Data-Efficient Molecular Generation with Hierarchical Textual Inversion [[Paper]](https://openreview.net/forum?id=wwotGBxtC3)
30. [2024 ACL] MolTC: Towards Molecular Relational Modeling In Language Models [[Paper]](https://arxiv.org/abs/2402.03781) [[Code]](https://github.com/MangoKiller/MolTC)
31. [2024 ACL Findings] ReactXT: Understanding Molecular Relationship via Reaction-Contextualized Molecule-Text Pretraining [[Paper]](https://arxiv.org/abs/2405.14225) [[Code]](https://github.com/syr-cn/ReactXT)
32. [2024 KDD] Learning Multi-View Molecular Representations with Structured and Unstructured Knowledge [[Paper]](https://arxiv.org/abs/2406.09841) [[Code]](https://github.com/PharMolix/OpenBioMed)
33. [2024 CIKM] Vision Language Model is NOT All You Need: Augmentation Strategies for Molecule Language Model [[Paper]](arxiv.org/abs/2407.09043) [[Code]](https://github.com/Namkyeong/AMOLE)


## Survey on Molecules and Language Models
1. [2023 arXiv] Transformers and Large Language Models for Chemistry and Drug Discovery [[Paper]](https://arxiv.org/abs/2310.06083)
2. [2023 arXiv] Language models in molecular discovery [[Paper]](https://arxiv.org/abs/2309.16235)
3. [2024 arXiv] Scientific Language Modeling: A Quantitative Review of Large Language Models in Molecular Science [[Paper]](https://arxiv.org/abs/2402.04119)


## Categories

- [Versatile Docking](#versatile-docking)
- [Protein-Ligand Docking](#protein-ligand-docking)
  - [2024 -- Protein-Ligand](#2024----protein-ligand)
  - [2023 -- Protein-Ligand](#2023----protein-ligand)
  - [2022 -- Protein-Ligand](#2022----protein-ligand)
- [Protein-Protein Docking](#protein-protein-docking)
  - [2024 -- Protein-Protein](#2024----protein-protein)
  - [2023 -- Protein-Protein](#2023----protein-protein)
  - [2022 -- Protein-Protein](#2022----protein-protein)
  - [2021 -- Protein-Protein](#2021----protein-protein)
- [Protein-Nucleic Acid Docking](#protein-nucleic-acid-docking)
  - [2023 -- Protein-Nucleic Acid](#2023----protein-nucleic-acid)
- [Covalent Docking](#covalent-docking)
- [Survey](#survey)
  - [Protein-Ligand](#protein-ligand)
  - [Protein-Protein](#protein-protein)
  - [Protein-Nucleic Acid](#protein-nucleic-acid)
  - [Covalent](#covalent)
- [Traditional Docking Tools](#traditional-docking-tools)
  - [Open-source and free access](#open-source-and-free-access)
  - [Commercial tools](#commercial-tool)

---

# Docking

## Versatile Docking

🔥**Chai-1 Technical Report**   
Chai-1 Discovery Team  
*Report, Sep 2024*  
[![](https://img.shields.io/badge/report-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://chaiassets.com/chai-1/paper/technical_report_v1.pdf)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/chaidiscovery/chai-lab/)
![Stars](https://img.shields.io/github/stars/chaidiscovery/chai-lab?color=yellow&style=social)

🔥 **AlphaFold3 Open-Source Implementation (Ligo)**   
Edward Harris, Emily Egerton-Warburton, Arda Goreci  
*Project, Sep 2024*  
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/Ligo-Biosciences/AlphaFold3)
![Stars](https://img.shields.io/github/stars/Ligo-Biosciences/AlphaFold3?color=yellow&style=social)


🔥**Accurate structure prediction of biomolecular interactions with AlphaFold 3**   
Josh Abramson, Jonas Adler, Jack Dunger, Richard Evans, Tim Green, Alexander Pritzel, Olaf Ronneberger, Lindsay Willmore, Andrew J. Ballard, Joshua Bambrick, Sebastian W. Bodenstein, David A. Evans, Chia-Chun Hung, Michael O’Neill, David Reiman, Kathryn Tunyasuvunakool, Zachary Wu, Akvilė Žemgulytė, Eirini Arvaniti, Charles Beattie, Ottavia Bertolli, Alex Bridgland, Alexey Cherepanov, Miles Congreve, Alexander I. Cowen-Rivers, Andrew Cowie, Michael Figurnov, Fabian B. Fuchs, Hannah Gladman, Rishub Jain, Yousuf A. Khan, Caroline M. R. Low, Kuba Perlin, Anna Potapenko, Pascal Savy, Sukhdeep Singh, Adrian Stecula, Ashok Thillaisundaram, Catherine Tong, Sergei Yakneen, Ellen D. Zhong, Michal Zielinski, Augustin Žídek, Victor Bapst, Pushmeet Kohli, Max Jaderberg, Demis Hassabis & John M. Jumper  
*Nature, May 2024*  
[![](https://img.shields.io/badge/nature-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.nature.com/articles/s41586-024-07487-w)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F7572ba7f604ef95d7acdd657ebac458106bd35df%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/news-AB9FF2?style=flat&logo=temporal&labelColor=555555)](https://blog.google/technology/ai/google-deepmind-isomorphic-alphafold-3-ai-model/)
[![](https://img.shields.io/badge/server-AB9FF2?style=flat&logo=temporal&labelColor=555555)](https://golgi.sandbox.google.com/)

🔥**Generalized Biomolecular Modeling and Design with RoseTTAFold All-Atom**  
Rohith Krishna, Jue Wang, Woody Ahern, Pascal Sturmfels, Preetham Venkatesh, Indrek Kalvet, Gyu Rie Lee, Felix S Morey-Burrows, Ivan Anishchenko, Ian R Humphreys, Ryan McHugh, Dionne Vafeados, Xinting Li, George A Sutherland, Andrew Hitchcock, C Neil Hunter, Minkyung Baek, Frank DiMaio, David Baker  
*Science, March 2024*  
[![](https://img.shields.io/badge/science-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.science.org/doi/10.1126/science.adl2528)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fbbd97deb6e06fe24c5f20fa85e1f276e3065f99f%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/baker-laboratory/RoseTTAFold-All-Atom)
![Stars](https://img.shields.io/github/stars/baker-laboratory/RoseTTAFold-All-Atom?color=yellow&style=social)

🔥**A glimpse of the next generation of AlphaFold**   
Google DeepMind AlphaFold team and Isomorphic Labs team  
*News, Oct 2023*  
[![](https://img.shields.io/badge/report-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://storage.googleapis.com/deepmind-media/DeepMind.com/Blog/a-glimpse-of-the-next-generation-of-alphafold/alphafold_latest_oct2023.pdf)
[![](https://img.shields.io/badge/news-AB9FF2?style=flat&logo=temporal&labelColor=555555)](https://deepmind.google/discover/blog/a-glimpse-of-the-next-generation-of-alphafold)
    


## Protein-Ligand Docking

### 2024 -- Protein-Ligand
**One-step Structure Prediction and Screening for Protein-Ligand Complexes using Multi-Task Geometric Deep Learning**  
Kelei He, Tiejun Dong, Jinhui Wu, Junfeng Zhang  
*Preprint, Aug 2024*  
[![](https://img.shields.io/badge/arXiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://arxiv.org/pdf/2408.11356)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://gitfront.io/r/LigPose/kMWuV4DW6JpE/LigPose4Review/)
![](https://img.shields.io/badge/known--pocket-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 


**Fully flexible molecular alignment enables accurate ligand structure modelling**  
Zhihao Wang, Fan Zhou, Zechen Wang, Qiuyue Hu, Yong-Qiang Li, Sheng Wang, Yanjie Wei, Liangzhen Zheng, Weifeng Li, Xiangda Peng  
*JCIM, July 2024*  
[![](https://img.shields.io/badge/JCIM-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://pubs.acs.org/doi/10.1021/acs.jcim.4c00669)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fbf416a733a88126a3c46b6c28788ba51b22f4846%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)


**Deep Learning for Protein-Ligand Docking: Are We There Yet?**  
Alex Morehead, Nabin Giri, Jian Liu, Jianlin Cheng  
*Preprint, May 2024*  
[![](https://img.shields.io/badge/arXiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://arxiv.org/pdf/2405.14108)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/BioinfoMachineLearning/PoseBench)
![Stars](https://img.shields.io/github/stars/BioinfoMachineLearning/PoseBench?color=yellow&style=social)
![](https://img.shields.io/badge/dataset-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 


**Uni-Mol Docking V2: Towards Realistic and Accurate Binding Pose Prediction**  
Eric Alcaide, Zhifeng Gao, Guolin Ke, Yaqi Li, Linfeng Zhang, Hang Zheng, Gengmo Zhou  
*Preprint, May 2024*  
[![](https://img.shields.io/badge/arXiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://arxiv.org/pdf/2405.11769)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fd03bcd5003d5f35d536f9e9b7ae5c4194de93295%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/dptech-corp/Uni-Mol)
![Stars](https://img.shields.io/github/stars/dptech-corp/Uni-Mol?color=yellow&style=social)
![](https://img.shields.io/badge/known--pocket-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 


**CarsiDock: a deep learning paradigm for accurate protein–ligand docking and screening based on large-scale pre-training**  
Heng Cai, Chao Shen, Tianye Jian, Xujun Zhang, Tong Chen, Xiaoqi Han, Zhuo Yang, Wei Dang, Chang-Yu Hsieh, Yu Kang, Peichen Pan, Xiangyang Ji, Jianfei Song, Tingjun Hou and Yafeng Deng  
*Chemical Science, December 2023*  
[![](https://img.shields.io/badge/chemical_science-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://pubs.rsc.org/en/content/articlepdf/2024/sc/d3sc05552c)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F12f90a54c1f94981ba4f129be8b220b4b8282a6b%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/carbonsilicon-ai/CarsiDock)
![Stars](https://img.shields.io/github/stars/carbonsilicon-ai/CarsiDock?color=yellow&style=social)
![](https://img.shields.io/badge/known--pocket-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**GeoDirDock: Guiding Docking Along Geodesic Paths**  
Raúl Miñán, Javier Gallardo, Álvaro Ciudad, Alexis Molina  
*Preprint, April 2024*  
[![](https://img.shields.io/badge/under_review-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://arxiv.org/pdf/2404.06481.pdf)

**Interformer: An Interaction-Aware Model for Protein-Ligand Docking and Affinity Prediction**  
Houtim Lai, Longyue Wang, Ruiyuan Qian, Geyan Ye, Juhong Huang, Fandi Wu, Fang Wu, Xiangxiang Zeng, Wei Liu  
*Preprint, April 2024*  
[![](https://img.shields.io/badge/under_review-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.researchsquare.com/article/rs-3995849/v1)

**FABind+: Enhancing Molecular Docking through Improved Pocket Prediction and Pose Generation**  
Kaiyuan Gao, Qizhi Pei, Jinhua Zhu, Tao Qin, Kun He, Lijun Wu  
*Preprint, April 2024*  
[![](https://img.shields.io/badge/arXiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://arxiv.org/pdf/2403.20261.pdf)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F6594ec85ba41682c2ae0b2c205a92ec372f0ec4b%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/QizhiPei/FABind)
![Stars](https://img.shields.io/github/stars/QizhiPei/FABind?color=yellow&style=social)
![](https://img.shields.io/badge/blind-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**ArtiDock: fast and accurate machine learning approach to protein-ligand docking based on multimodal data augmentation**  
Taras Voitsitskyi, Semen Yesylevskyy, Volodymyr Bdzhola, Roman Stratiichuk, Ihor Koleiev, Zakhar Ostrovsky, Volodymyr Vozniak, Ivan Khropachov, Pavlo Henitsoi, Leonid Popryho, Roman Zhytar, Alan Nafiiev, Serhii Starosyla  
*Preprint, March 2024*  
[![](https://img.shields.io/badge/biorxiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.biorxiv.org/content/10.1101/2024.03.14.585019)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fad788c56812bea808f4cb5c31f208a6347f1d24d%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/news-AB9FF2?style=flat&logo=temporal&labelColor=555555)](https://www.biopharmatrend.com/post/713-artidock-from-receptorai-next-generation-ai-docking-that-beats-diffdock-and-alphafold-latest)
![](https://img.shields.io/badge/known--pocket-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**Re-Dock: Towards Flexible and Realistic Molecular Docking with Diffusion Bridge**  
Yufei Huang, Odin Zhang, Lirong Wu, Cheng Tan, Haitao Lin, Zhangyang Gao, Siyuan Li, Stan. Z. Li  
*Preprint, February 2024*  
[![](https://img.shields.io/badge/arxiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://arxiv.org/pdf/2402.11459.pdf)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F68981c9aba4c06176f3c062b94c3e6861371bdb6%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
![](https://img.shields.io/badge/blind-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 
![](https://img.shields.io/badge/flexible-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**PackDock: a Diffusion Based Side Chain Packing Model for Flexible Protein-Ligand Docking**  
Runze Zhang, Xinyu Jiang, Duanhua Cao, Jie Yu, Mingan Chen, Zhehuan Fan, Xiangtai Kong, Jiacheng Xiong, Zimei Zhang, Wei Zhang, Shengkun Ni, Yitian Wang, Shenghua Gao, Mingyue Zheng  
*Preprint, February 2024*  
[![](https://img.shields.io/badge/biorxiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.biorxiv.org/content/10.1101/2024.01.31.578200v1.full.pdf)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F770f53f09f541b96d0e3693ae4066b897ef9d9f5%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/Zhang-Runze/PackDock)
![Stars](https://img.shields.io/github/stars/Zhang-Runze/PackDock?color=yellow&style=social)
![](https://img.shields.io/badge/known--pocket-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 
![](https://img.shields.io/badge/flexible-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**State-specific protein–ligand complex structure prediction with a multiscale deep generative model**  
Zhuoran Qiao, Weili Nie, Arash Vahdat, Thomas F. Miller II, Animashree Anandkumar  
*Nature Machine Intelligence, February 2024*  
[![](https://img.shields.io/badge/nature-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.nature.com/articles/s42256-024-00792-z)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fc9561b15c25ca781ccf1a384e978e28341bac0e4%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/zrqiao/NeuralPLexer)
![Stars](https://img.shields.io/github/stars/zrqiao/NeuralPLexer?color=yellow&style=social)
![](https://img.shields.io/badge/blind-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 
![](https://img.shields.io/badge/flexible-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**DynamicBind: Predicting ligand-specific protein-ligand complex structure with a deep equivariant generative model**  
Wei Lu, Jixian Zhang, Huang Weifeng, Ziqiao Zhang, Chengtao Li, Shuangjia Zheng  
*Nature Communications, February 2024*  
[![](https://img.shields.io/badge/nature-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.nature.com/articles/s41467-024-45461-2)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F51894ffcf778630e7b021bc473d13d028a3b9158%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/luwei0917/DynamicBind)
![Stars](https://img.shields.io/github/stars/luwei0917/DynamicBind?color=yellow&style=social)
![](https://img.shields.io/badge/blind-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 
![](https://img.shields.io/badge/flexible-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**Deep confident steps to new pockets: strategies for docking generalization**   
Gabriele Corso, Arthur Deng, Nicholas Polizzi, Regina Barzilay, Tommi S. Jaakkola   
*ICLR, Feburary 2024*   
[![](https://img.shields.io/badge/iclr-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://openreview.net/forum?id=UfBIxpTK10) 
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fa2af79d2c6a75919d586590487cc3b39b42a94c2%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/LDeng0205/confidence-bootstrapping)
![Stars](https://img.shields.io/github/stars/LDeng0205/confidence-bootstrapping?color=yellow&style=social)
![](https://img.shields.io/badge/blind-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**(NeuralMD) A Multi-Grained Symmetric Differential Equation Model for Learning Protein-Ligand Binding Dynamics**  
Shengchao Liu, Weitao Du, Yanjing Li, Zhuoxinran Li, Vignesh Bhethanabotla, Nakul Rampal, Omar Yaghi, Christian Borgs, Anima Anandkumar, Hongyu Guo, Jennifer Chayes  
*Preprint, January 2024*  
[![](https://img.shields.io/badge/arxiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://arxiv.org/pdf/2401.15122.pdf) 
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F0215dd9f346534bf4c4247220501d7ab7d7715c6%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
![](https://img.shields.io/badge/dynamics-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

### 2023 -- Protein-Ligand 
**(DeltaDock) Multi-scale Iterative Refinement towards Robust and Versatile Molecular Docking**  
Jiaxian Yan, Zaixi Zhang, Kai Zhang, Qi Liu  
*Preprint, December 2023*  
[![](https://img.shields.io/badge/arxiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://arxiv.org/abs/2311.18574)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fbedc29f7b553bd322df265e2c5a215f1dfd19b4e%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
![](https://img.shields.io/badge/blind-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**DiffBindFR: An SE(3) Equivariant Network for Flexible Protein-Ligand Docking**  
Jintao Zhu, Zhonghui Gu, Jianfeng Pei, Luhua Lai  
*Preprint, November 2023*
[![](https://img.shields.io/badge/biorxiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://arxiv.org/pdf/2311.15201.pdf)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F04911aa5c3b5d6d07507a0079ebbe6c504a462ae%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/HBioquant/DiffBindFR)
![Stars](https://img.shields.io/github/stars/HBioquant/DiffBindFR?color=yellow&style=social)
![](https://img.shields.io/badge/known--pocket-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 
![](https://img.shields.io/badge/flexible-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**Structure prediction of protein-ligand complexes from sequence information with Umol**  
Patrick Bryant, Atharva Kelkar, Andrea Guljas, Cecilia Clementi, Frank Noé  
*Preprint, November 2023*  
[![](https://img.shields.io/badge/biorxiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://pubmed.ncbi.nlm.nih.gov/37745556/)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F3660911f955c532c186de3b52b03396c851c0aac%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/patrickbryant1/Umol)
![Stars](https://img.shields.io/github/stars/patrickbryant1/Umol?color=yellow&style=social)
![](https://img.shields.io/badge/known--pocket-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**(FlexPose) Equivariant Flexible Modeling of the Protein–Ligand Binding Pose with Geometric Deep Learning**  
Tiejun Dong, Ziduo Yang, Jun Zhou, and Calvin Yu-Chian Chen  
*JCTC, November 2023*  
[![](https://img.shields.io/badge/jctc-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://pubs.acs.org/doi/full/10.1021/acs.jctc.3c00273)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Ff2633ad706a8537bd7d8d89561d4bbad71373380%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/tiejundong/FlexPose)
![Stars](https://img.shields.io/github/stars/tiejundong/FlexPose?color=yellow&style=social)
![](https://img.shields.io/badge/known--pocket-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 
![](https://img.shields.io/badge/flexible-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**Pre-Training on Large-Scale Generated Docking Conformations with HelixDock to Unlock the Potential of Protein-ligand Structure Prediction Models**  
Lihang Liu, Shanzhuo Zhang, Donglong He, Xianbin Ye, Jingbo Zhou, Xiaonan Zhang, Yaoyao Jiang, Weiming Diao, Hang Yin, Hua Chai, Fan Wang, Jingzhou He, Liang Zheng, Yonghui Li, Xiaomin Fang  
*Preprint, October 2023*  
[![](https://img.shields.io/badge/arxiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://arxiv.org/abs/2310.13913)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fe0b05305ce09e20d64394753ac2eca2f7aaf7d9d%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/PaddlePaddle/PaddleHelix/tree/dev/apps/molecular_docking/helixdock)
![Stars](https://img.shields.io/github/stars/PaddlePaddle/PaddleHelix?color=yellow&style=social)

**PoseBusters: AI-based docking methods fail to generate physically valid poses or generalise to novel sequences**  
Martin Buttenschoen, Garrett M. Morris, Charlotte M. Deane  
*Preprint, October 2023.*  
[![](https://img.shields.io/badge/arxiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://arxiv.org/abs/2308.05777)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fdd267334c01365d362ed8239394f0ad61b41e269%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/maabuu/posebusters)
![Stars](https://img.shields.io/github/stars/maabuu/posebusters?color=yellow&style=social)
![](https://img.shields.io/badge/dataset-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**FABind: Fast and Accurate Protein-Ligand Binding**  
Qizhi Pei, Kaiyuan Gao, Lijun Wu, Jinhua Zhu, Yingce Xia, Shufang Xie, Tao Qin, Kun He, Tie-Yan Liu, Rui Yan  
*NeurIPS, September 2023*  
[![](https://img.shields.io/badge/neurips-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://openreview.net/forum?id=PnWakgg1RL)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fef5fceee2925cb8441bf1de100b67a33eeeef3a3%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/QizhiPei/FABind)
![Stars](https://img.shields.io/github/stars/QizhiPei/FABind?color=yellow&style=social)
![](https://img.shields.io/badge/blind-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**Efficient and accurate large library ligand docking with KarmaDock**  
Xujun Zhang, Odin Zhang, Chao Shen, Wanglin Qu, Shicheng Chen, Hanqun Cao, Yu Kang, Zhe Wang, Ercheng Wang, Jintu Zhang, Yafeng Deng, Furui Liu, Tianyue Wang, Hongyan Du, Langcheng Wang, Peichen Pan, Guangyong Chen, Chang-Yu Hsieh, Tingjun Hou  
*Nature Computational Science, September 2023*  
[![](https://img.shields.io/badge/nature-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://doi.org/10.1038/s43588-023-00511-5)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F190320029c58f7e3c1ce8fa9b908d3c88a27df2f%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/schrojunzhang/KarmaDock)
![Stars](https://img.shields.io/github/stars/schrojunzhang/KarmaDock?color=yellow&style=social)
![](https://img.shields.io/badge/known--pocket-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**(EDM-Dock) Deep Learning Model for Efficient Protein–Ligand Docking with Implicit Side-Chain Flexibility**  
Matthew R. Masters, Amr H. Mahmoud, Yao Wei, and Markus A. Lill  
*JCIM, March 2023*  
[![](https://img.shields.io/badge/nature-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://doi.org/10.1038/s43588-023-00511-5)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fb29d66e1c466bd248232eee0d987118b258fb18e%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/MatthewMasters/EDM-Dock)
![Stars](https://img.shields.io/github/stars/MatthewMasters/EDM-Dock?color=yellow&style=social)
![](https://img.shields.io/badge/known--pocket-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 
![](https://img.shields.io/badge/flexible-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**Do deep learning models really outperform traditional approaches in molecular docking?**  
Yuejiang Yu, Shuqi Lu, Zhifeng Gao, Hang Zheng, Guolin Ke  
*ICLR workshop MLDD, March 2023*  
[![](https://img.shields.io/badge/iclr_workshop-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://openreview.net/forum?id=JrtHZdbGtN)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F6fd91f8c48b273bb6c256d5ed4e250edd1b156fa%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
![](https://img.shields.io/badge/blind-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

🔥**DiffDock: Diffusion Steps, Twists, and Turns for Molecular Docking**   
Gabriele Corso, Hannes Stärk, Bowen Jing, Regina Barzilay, Tommi Jaakkola  
*ICLR, Feburary 2023*  
[![](https://img.shields.io/badge/iclr-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://openreview.net/forum?id=kKF8_K-mBbS)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F6f0d0b897d0e0963204719b80a8af43ca0d79d90%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/gcorso/DiffDock)
![Stars](https://img.shields.io/github/stars/gcorso/DiffDock?color=yellow&style=social)
![](https://img.shields.io/badge/blind-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**Uni-Mol: A Universal 3D Molecular Representation Learning Framework**  
Gengmo Zhou, Zhifeng Gao, Qiankun Ding, Hang Zheng, Hongteng Xu, Zhewei Wei, Linfeng Zhang, Guolin Ke  
*ICLR, Feburary 2023*  
[![](https://img.shields.io/badge/iclr-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://openreview.net/forum?id=6K2RM6wVqKu)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F11f42721f56f36a64638677ccde7784040829656%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/dptech-corp/Uni-Mol)
![Stars](https://img.shields.io/github/stars/dptech-corp/Uni-Mol?color=yellow&style=social)
![](https://img.shields.io/badge/known--pocket-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**E3Bind: An End-to-End Equivariant Network for Protein-Ligand Docking**  
Yangtian Zhang, Huiyu Cai, Chence Shi, Jian Tang  
*ICLR, Feburary 2023*  
[![](https://img.shields.io/badge/iclr-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://openreview.net/forum?id=sO1QiAftQFv)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F7cfc990d89875342528b760a4ffed9de47010b03%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
![](https://img.shields.io/badge/blind-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

### 2022 -- Protein-Ligand 

**TANKBind: Trigonometry-Aware Neural NetworKs for Drug-Protein Binding Structure Prediction**  
Wei Lu, Qifeng Wu, Jixian Zhang, Jiahua Rao, Chengtao Li, Shuangjia Zheng  
*NeurIPS, November 2022*  
[![](https://img.shields.io/badge/neurips-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://openreview.net/forum?id=MSBDFwGYwwt)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fb886797dc34c91f186629403d7c7e2092fc25083%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/luwei0917/TankBind)
![Stars](https://img.shields.io/github/stars/luwei0917/TankBind?color=yellow&style=social)
![](https://img.shields.io/badge/blind-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**EquiBind: Geometric Deep Learning for Drug Binding Structure Prediction**  
Hannes Stärk, Octavian Ganea, Lagnajit Pattanaik, Dr.Regina Barzilay, Tommi Jaakkola  
*ICML, July 2022*  
[![](https://img.shields.io/badge/neurips-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://proceedings.mlr.press/v162/stark22b.html)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F5309f4bcb15e3dafbed759488551c1650b55dd81%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/HannesStark/EquiBind)
![Stars](https://img.shields.io/github/stars/HannesStark/EquiBind?color=yellow&style=social)
![](https://img.shields.io/badge/blind-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

## Protein-Protein Docking
### 2024 -- Protein-Protein
**Deep Reinforcement Learning for Modelling Protein Complexes**  
Ziqi Gao, Tao Feng, Jiaxuan You, Chenyi Zi, Yan Zhou, Chen Zhang, Jia Li  
*ICLR, March 2024*  
[![](https://img.shields.io/badge/iclr-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://arxiv.org/pdf/2405.02299)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F5301ef92d5d9d5f2ae0771db8e5b1ea66ea96788%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
![](https://img.shields.io/badge/assembly-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**Improving deep learning protein monomer and complex structure prediction using DeepMSA2 with huge metagenomics data**  
Wei Zheng, Qiqige Wuyun, Yang Li, Chengxin Zhang, P. Lydia Freddolino, Yang Zhang  
*Nature Methods, January 2024*  
[![](https://img.shields.io/badge/nature-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.nature.com/articles/s41592-023-02130-4)
[![](https://img.shields.io/badge/deepmsa2-AB9FF2?style=flat&logo=temporal&labelColor=555555&logocolor=555555)](https://zhanggroup.org/DeepMSA)
[![](https://img.shields.io/badge/dmfold-AB9FF2?style=flat&logo=temporal&labelColor=555555&logocolor=555555)](https://zhanggroup.org/DMFold)

**Neural Probabilistic Protein-Protein Docking via a Differentiable Energy Model**  
Huaijin Wu, Wei Liu, Yatao Bian, Jiaxiang Wu, Nianzu Yang, Junchi Yan  
*ICLR, March 2024*  
[![](https://img.shields.io/badge/iclr-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://openreview.net/forum?id=qg2boc2AwU) 
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/wuhuaijin/EBMDock) 
![Stars](https://img.shields.io/github/stars/wuhuaijin/EBMDock?color=yellow&style=social) 
![](https://img.shields.io/badge/rigid-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**Rigid Protein-Protein Docking via Equivariant Elliptic-Paraboloid Interface Prediction**  
Ziyang Yu, Wenbing Huang, Yang Liu  
*ICLR, March 2024*  
[![](https://img.shields.io/badge/iclr-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://openreview.net/forum?id=zgQ0PHeGnL)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F7868daf3774809e652f83914fb7b44e36a78d7f2%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/yaledeus/ElliDock)
![Stars](https://img.shields.io/github/stars/yaledeus/ElliDock?color=yellow&style=social)
![](https://img.shields.io/badge/rigid-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

### 2023 -- Protein-Protein
**(GeoDock) Flexible protein–protein docking with a multitrack iterative transformer**  
Lee-Shin Chu, Jeffrey A. Ruffolo, Ameya Harmalkar, Jeffrey J. Gray  
*Protein Science, December 2023*  
[![](https://img.shields.io/badge/prosci-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://onlinelibrary.wiley.com/doi/epdf/10.1002/pro.4862)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F1c8af104ce6af12952581007aa19baa63f663a8f%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/Graylab/GeoDock)
![Stars](https://img.shields.io/github/stars/Graylab/GeoDock?color=yellow&style=social)

**Enhancing alphafold-multimer-based protein complex structure prediction with MULTICOM in CASP15**  
Jian Liu, Zhiye Guo, Tianqi Wu, Rajashree Roy, Farhan Quadir, Chen Chen, Jianlin Cheng  
*Communications Biology, November 2023*  
[![](https://img.shields.io/badge/Communications_Biology-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.nature.com/articles/s42003-023-05525-3)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F679d7389ab657a666d0149cfad9ce3b62ee50870%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/BioinfoMachineLearning/MULTICOM3)
![Stars](https://img.shields.io/github/stars/BioinfoMachineLearning/MULTICOM3?color=yellow&style=social)
![](https://img.shields.io/badge/add--on_for_alphafold-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555)  

**DockGame: Cooperative Games for Multimeric Rigid Protein Docking**  
Vignesh Ram Somnath, Pier Giuseppe Sessa, Maria Rodriguez Martinez, Andreas Krause  
*Preprint, October 2023*  
[![](https://img.shields.io/badge/arXiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://arxiv.org/abs/2310.06177)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F0a76e5107737cd3017a880f52b91ccade4504bf3%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/vsomnath/dockgame)
![Stars](https://img.shields.io/github/stars/vsomnath/dockgame?color=yellow&style=social)
![](https://img.shields.io/badge/rigid-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 
![](https://img.shields.io/badge/multimeric-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555)

**Diffdock-pp: Rigid protein-protein docking with diffusion models**  
Mohamed Amine Ketata, Cedrik Laue, Ruslan Mammadov, Hannes Stärk, Menghua Wu, Gabriele Corso, Céline Marquet, Regina Barzilay, Tommi S. Jaakkola  
*ICLR workshop MLDD, March 2023*  
[![](https://img.shields.io/badge/iclr_workshop-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://openreview.net/forum?id=AM7WbQxuRS)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F345eb3195aac43eb5dcaacc0a7fd288be0fa4491%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/ketatam/DiffDock-PP)
![Stars](https://img.shields.io/github/stars/ketatam/DiffDock-PP?color=yellow&style=social)
![](https://img.shields.io/badge/rigid-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**Deep Learning for Flexible and Site-Specific Protein Docking and Design**  
Matthew McPartlon, Jinbo Xu  
*BioRxiv, April 2023*  
[![](https://img.shields.io/badge/bioRxiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.biorxiv.org/content/10.1101/2023.04.01.535079)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F663d7f9d2b4b8fcbb488fdb07a7458f734e62726%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
![](https://img.shields.io/badge/flexible-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 
![](https://img.shields.io/badge/known--pocket-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555)  

### 2022 -- Protein-Protein
**Physics-informed deep neural network for rigid-body protein docking**  
Freyr Sverrisson, Jean Feydy, Joshua Southern, Michael M Bronstein, Bruno E Correia  
*ICLR workshop MLDD, April 2022*  
[![](https://img.shields.io/badge/iclr_workshop-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://openreview.net/forum?id=5yn5shS6wN)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F59f1e3f333f85db9294658ccfe7f0a5f6a9458bd%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
![](https://img.shields.io/badge/rigid-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555)  

**Independent SE(3)-Equivariant Models for End-to-End Rigid Protein Docking**  
Octavian-Eugen Ganea, Xinyuan Huang, Charlotte Bunne, Yatao Bian, Regina Barzilay, Tommi S. Jaakkola, Andreas Krause  
*ICLR, January 2022*  
[![](https://img.shields.io/badge/iclr-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://openreview.net/forum?id=GQjaI9mLet)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fb190df7856f71856cbd98e1394c6513df279d53e%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/octavian-ganea/equidock_public)
![Stars](https://img.shields.io/github/stars/octavian-ganea/equidock_public?color=yellow&style=social)
![](https://img.shields.io/badge/rigid-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

### 2021 -- Protein-Protein 
🔥**Protein complex prediction with AlphaFold-Multimer**  
Richard Evans, Michael O’Neill, A. Pritzel, Natasha Antropova, Andrew Senior, Tim Green, Augustin Zídek, Russ Bates, Sam Blackwell, Jason Yim, O. Ronneberger, S. Bodenstein, Michal Zielinski, Alex Bridgland, Anna Potapenko, Andrew Cowie, Kathryn Tunyasuvunakool, Rishub Jain, Ellen Clancy, Pushmeet Kohli, J. Jumper, D. Hassabis  
*BioRxiv, October 2021*  
[![](https://img.shields.io/badge/bioRxiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.biorxiv.org/content/10.1101/2021.10.04.463034)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F2556e820cba6bda75f6f31b76bc74d9e36d72cb3%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/google-deepmind/alphafold)
![Stars](https://img.shields.io/github/stars/google-deepmind/alphafold?color=yellow&style=social) 
![](https://img.shields.io/badge/flexible-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555)  

## Protein-Nucleic Acid Docking
### 2023 -- Protein-Nucleic Acid

🔥**Accurate prediction of protein-nucleic acid complexes using RoseTTAFoldNA**  
Minkyung Baek, Ryan McHugh, Ivan Anishchenko, Hanlun Jiang, David Baker, Frank DiMaio  
*Nature Methods, November 2023*  
[![](https://img.shields.io/badge/nature-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.nature.com/articles/s41592-023-02086-5)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F0c0ad28901d1cae2cc6c7b9c1f66e9e46afc5ec5%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/uw-ipd/RoseTTAFold2NA)
![Stars](https://img.shields.io/github/stars/uw-ipd/RoseTTAFold2NA?color=yellow&style=social)

**EquiPNAS: improved protein-nucleic acid binding site prediction using protein-language-model-informed equivariant deep graph neural networks**  
Rahmatullah Roche, Bernard Moussad, Md Hossain Shuvo, Sumit Tarafder, Debswapna Bhattacharya  
*BioRxiv, September 2023*  
[![](https://img.shields.io/badge/bioRxiv-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.biorxiv.org/search/Protein-Nucleic%252BAcid%252B)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/Bhattacharya-Lab/EquiPNAS)
![Stars](https://img.shields.io/github/stars/Bhattacharya-Lab/EquiPNAS?color=yellow&style=social)
![](https://img.shields.io/badge/language_model-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**Evaluating native-like structures of RNA-protein complexes through the deep learning method**  
Chengwei Zeng, Yiren Jian, Soroush Vosoughi, Chen Zeng, Yunjie Zhao  
*Nature Communications, February 2023*  
[![](https://img.shields.io/badge/nature-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.nature.com/articles/s41467-023-36720-9)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F43253ab8bfa9f19c18764fdcfb550395f5cdfc75%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/Zhaolab-GitHub/DRPScore_v1.0)
![Stars](https://img.shields.io/github/stars/Zhaolab-GitHub/DRPScore_v1.0?color=yellow&style=social)
![](https://img.shields.io/badge/evaluation-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 


## Covalent Docking

**Cov_DOX: A Method for Structure Prediction of Covalent Protein–Ligand Bindings**  
Lin Wei, Yaru Chen, Jiaqi Liu, Li Rao, Yanliang Ren, Xin Xu, Jian Wan  
*Journal of Medicinal Chemistry, March 2022*  
[![](https://img.shields.io/badge/Medicinal_Chemistry-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c02007)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2FURL%3Ahttps%3A%2F%2Fwww.semanticscholar.org%2Fpaper%2FCov_DOX%253A-A-Method-for-Structure-Prediction-of-Wei-Chen%2Fec80310e4d4a9f0308963e1cfccbd624d9fde2da%2F%3Ffields%3DcitationCount&query=citationCount&style=social&label=Citation&labelColor=555555&color=ED8936)
![](https://img.shields.io/badge/non--cov_based-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555)  

**CovPDB: a high-resolution coverage of the covalent protein–ligand interactome**  
Mingjie Gao, Aurelien F. A. Moumbock, Ammar Qaseem, Qianqing Xu, Stefan Gunther  
*Nucleic Acids Research, September 2021*  
[![](https://img.shields.io/badge/Nucleic_Acids_Res.-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://academic.oup.com/nar/article/50/D1/D445/6377397)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2FURL%3Ahttps%3A%2F%2Fwww.semanticscholar.org%2Fpaper%2FCovPDB%253A-a-high-resolution-coverage-of-the-covalent-Gao-Moumbock%2Fa800cdd78504470a4b8c3a5fd31879a249686676%2F%3Ffields%3DcitationCount&query=citationCount&style=social&label=Citation&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/website-AB9FF2?style=flat&logo=temporal&labelColor=555555&logocolor=555555)](https://drug-discovery.vm.uni-freiburg.de/covpdb/)
![](https://img.shields.io/badge/dataset-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

**Fragment-based covalent ligand discovery**  
Wenchao Lu, Milka Kostic, Tinghu Zhang, Jianwei Che, Matthew P. Patricelli, Lyn H. Jones, Edward T. Chouchaniae, Nathanael S. Gray  
*RSC Chemical Biology, February 2021*  
[![](https://img.shields.io/badge/RSC_Chemical_Biology-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://pubs.rsc.org/en/content/articlelanding/2021/CB/D0CB00222D)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2FURL%3Ahttps%3A%2F%2Fwww.semanticscholar.org%2Fpaper%2FFragment-based-covalent-ligand-discovery-Lu-Kostic%2Fb525ba2c0b80e4587f2b1c6f667cf0009a46b356%2F%3Ffields%3DcitationCount&query=citationCount&style=social&label=Citation&labelColor=555555&color=ED8936)
![](https://img.shields.io/badge/hybrid_method-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555)  

**Covalent docking of large libraries for the discovery of chemical probes**  
Nir London, Rand M Miller, Shyam Krishnan, Kenji Uchida, John J Irwin, Oliv Eidam, Lucie Gibold, Peter Cimermančič, Richard Bonnet, Brian K Shoichet, Jack Taunton  
*Nature Chemical Biology, September 2014*  
[![](https://img.shields.io/badge/Nature_Chemical_Biology-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4232467/)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2FURL%3Ahttps%3A%2F%2Fwww.semanticscholar.org%2Fpaper%2FCovalent-Docking-of-Large-Libraries-for-the-of-London-Miller%2F90211028d92ba511542f8b1ccdbdbc7720e6710a%2F%3Ffields%3DcitationCount&query=citationCount&style=social&label=Citation&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/website-AB9FF2?style=flat&logo=temporal&labelColor=555555&logocolor=555555)](https://drug-discovery.vm.uni-freiburg.de/covpdb/)
![](http://covalent.docking.org/)
![](https://img.shields.io/badge/non--cov_based-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555)  

**Docking Covalent Inhibitors: A Parameter Free Approach To Pose Prediction and Scoring**  
Kai Zhu, Kenneth W. Borrelli, Jeremy R. Greenwood, Tyler Day, Robert Abel, Ramy S. Farid, and Edward Harder  
*Journal of Chemical Information and Modeling, June 2014*  
[![](https://img.shields.io/badge/Chemical_Information_and_Modeling-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://pubs.acs.org/doi/10.1021/ci500118s)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2FURL%3Ahttps%3A%2F%2Fwww.semanticscholar.org%2Fpaper%2FDocking-Covalent-Inhibitors%253A-A-Parameter-Free-To-Zhu-Borrelli%2F2d9998235f41a9b9ef91cde87dc1bab05b194592%2F%3Ffields%3DcitationCount&query=citationCount&style=social&label=Citation&labelColor=555555&color=ED8936)
![](https://img.shields.io/badge/parameter_free-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555)  

**CovalentDock: Automated covalent docking with parameterized covalent linkage energy estimation and molecular geometry constraints**  
Xuchang Ouyang, Shuo Zhou, Chinh Tran To Su, Zemei Ge, Runtao Li, Chee Keong Kwoh  
*Journal of Computational Chemistry, February 2013*  
[![](https://img.shields.io/badge/Computational_Chemistry-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://onlinelibrary.wiley.com/doi/10.1002/jcc.23136)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2FURL%3Ahttps%3A%2F%2Fwww.semanticscholar.org%2Fpaper%2FCovalentDock%253A-Automated-covalent-docking-with-and-Ouyang-Zhou%2F13e5c154c51d9d9950b8eca7668d4e83b957c1ae%2F%3Ffields%3DcitationCount&query=citationCount&style=social&label=Citation&labelColor=555555&color=ED8936)
[![](https://img.shields.io/badge/project-38C26D?style=flat&logo=googlecloud&labelColor=555555)](https://code.google.com/archive/p/covalentdock/)
![](https://img.shields.io/badge/Autodock_based-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 

## Survey
### Protein-Ligand 

**Machine-learning methods for ligand–protein molecular docking**  
Kevin Crampon, Alexis Giorkallos, Myrtille Deldossi, Stéphanie Baud, Luiz Angelo Steffenel   
*Drug Discovery Today, January 2022*  
[![](https://img.shields.io/badge/drugdis-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.sciencedirect.com/science/article/abs/pii/S1359644621003974)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F790e0a160fff355ce2fbb51beb4362dfcc58a830%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)

**A practical guide to large-scale docking**  
Brian J. Bender, Stefan Gahbauer, Andreas Luttens, Jiankun Lyu, Chase M. Webb, Reed M. Stein, Elissa A. Fink, Trent E. Balius, Jens Carlsson, John J. Irwin & Brian K. Shoichet    
*Nature Protocols, December 2021*  
[![](https://img.shields.io/badge/natureprotocols-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.nature.com/articles/s41596-021-00597-z)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F31a2f29489f4a95ed85f8ffb854de758e8be4c09%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)

**An Overview of Scoring Functions Used for Protein–Ligand Interactions in Molecular Docking**  
Jin Li, Ailing Fu, Le Zhang  
*Interdisciplinary Sciences: Computational Life Sciences, March 2019*  
[![](https://img.shields.io/badge/InterSciComLifeSci-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://link.springer.com/article/10.1007/s12539-019-00327-w)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F7eaa28830e31289a102084653654c3f911f05413%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)

**Progress in molecular docking**  
Jiyu Fan, Ailing Fu, Le Zhang  
*Quantitative Biology, June 2019*  
[![](https://img.shields.io/badge/QuantBio-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://link.springer.com/article/10.1007/s40484-019-0172-y)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F3c8b953cccb19a0a1679c28f73b33df20b2fd10e%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)

**Molecular Docking: Shifting Paradigms in Drug Discovery**  
Luca Pinzi, Giulio Rastelli    
*International Journal of Molecular Sciences, September 2019*  
[![](https://img.shields.io/badge/ijms-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.mdpi.com/1422-0067/20/18/4331)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F8f7948d72b19b3be7191396c5e637cdb14a2371c%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)

**From machine learning to deep learning: Advances in scoring functions for protein–ligand docking**  
Chao Shen, Junjie Ding, Zhe Wang, Dongsheng Cao, Xiaoqin Ding, Tingjun Hou   
*WIREs computational molecular science, June 2019*  
[![](https://img.shields.io/badge/ComMolSci-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://wires.onlinelibrary.wiley.com/doi/abs/10.1002/wcms.1429)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F9a297e83ca2bf1b3af089e51d85e927544ddfe04%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)

**Software for molecular docking: a review**  
Nataraj S. Pagadala, Khajamohiddin Syed, Jack Tuszynski   
*Biophysical Reviews, January 2017*  
[![](https://img.shields.io/badge/BioReview-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://link.springer.com/article/10.1007/s12551-016-0247-1)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fcd6e2fde7c80146a5f42bbe4e3638951e796ce0a%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)

**Dynamic Docking: A Paradigm Shift in Computational Drug Discovery**  
Gioia, Dario, Martina Bertazzo, Maurizio Recanatini, Matteo Masetti, Andrea Cavalli   
*Molecules, November 2017*  
[![](https://img.shields.io/badge/Molecules-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.mdpi.com/1420-3049/22/11/2029)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F49d40250c1bf1eb95b5c8caa3c2f663f336f117d%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)

**Insights into Protein–Ligand Interactions: Mechanisms, Models, and Methods**  
Xing Du, Yi Li, Yuan-Ling Xia, Shi-Meng Ai, Jing Liang, Peng Sang, Xing-Lai Ji, Shu-Qun Liu  
*International Journal of Molecular Sciences, January 2016*  
[![](https://img.shields.io/badge/MolSci-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.mdpi.com/1422-0067/17/2/144)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fbfd1b11cf32a063a7deec39ec0d76232f365582a%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)

### Protein-Protein

**Protein–Protein Docking: Past, Present, and Future**  
Sharon Sunny, PB Jayaraj  
*The protein journal, February 2022*  
[![](https://img.shields.io/badge/PROTEINJ-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://link.springer.com/article/10.1007/s10930-021-10031-8)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F60d7599aef518716e447853f0fa0cdafa774eabb%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)

**A survey on computational models for predicting protein–protein interactions**   
Lun Hu, Xiaojuan Wang, Yu-An Huang, Pengwei Hu, Zhu-Hong You  
*Briefings in Bioinformatics, September 2021*  
[![](https://img.shields.io/badge/BriefBioinform-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://academic.oup.com/bib/article/22/5/bbab036/6159365)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F171b6d5c35ac3cda7a821bff4041f360bee20e54%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)

**What method to use for protein–protein docking?**  
Kathryn Porter, Israel Desta, Dima Kozakov, Sandor Vajda  
*Current Opinion in Structural Biology, April 2019*   
[![](https://img.shields.io/badge/CURROPINSTRUCBIOL-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.sciencedirect.com/science/article/abs/pii/S0959440X18300691)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2F5098ef6ab4fbf1b5a7b180fd91cddac231abfd85%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)

### Protein-Nucleic Acid 

**Challenges in structural modeling of RNA-protein interactions**  
Xudong Liu, Yingtian Duan, Xu Hong, Juan Xie, Shiyong Liu  
*Current Opinion in Structural Biology, June 2023*  
[![](https://img.shields.io/badge/StructuralBiology-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://www.sciencedirect.com/science/article/pii/S0959440X23000970)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fb56c2784e1154c86541ee16b1aaeacfc6a46a321%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)

**Protein–RNA interaction prediction with deep learning: structure matters**  
Junkang Wei, Siyuan Chen, Licheng Zong, Xin Gao, Yu Li  
*Briefings in Bioinformatics， January 2022*  
[![](https://img.shields.io/badge/BrefInBio-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://academic.oup.com/bib/article/23/1/bbab540/6470965)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2Fd8eb3568107b21787751cd6c25776aacfa991979%3Ffields%3DcitationCount&query=%24.citationCount&label=citation&style=social&labelColor=555555&color=ED8936)

### Covalent 

**Docking covalent targets for drug discovery: stimulating the computer-aided drug design community of possible pitfalls and erroneous practices**  
Abdul-Quddus Kehinde Oyedele, Abdeen Tunde Ogunlana, Ibrahim Damilare Boyenle, Ayodeji Oluwadamilare Adeyemi, Temionu Oluwakemi Rita, Temitope Isaac Adelusi, Misbaudeen Abdul-Hammed, Oluwabamise Emmanuel Elegbeleye, Tope Tunji Odunitan  
*Molecular Diversity, September 2022*  
[![](https://img.shields.io/badge/Molecular_Diversity-5291C8?style=flat&logo=Read.cv&labelColor=555555)](https://link.springer.com/article/10.1007/s11030-022-10523-4)
![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.semanticscholar.org%2Fgraph%2Fv1%2Fpaper%2FURL%3Ahttps%3A%2F%2Fwww.semanticscholar.org%2Fpaper%2FDocking-covalent-targets-for-drug-discovery%253A-the-of-Oyedele-Ogunlana%2Ff347739ce3bbe75cf8bebe42aebbed12d4d9da6c%2F%3Ffields%3DcitationCount&query=citationCount&style=social&label=Citation&labelColor=555555&color=ED8936)

## Traditional Docking Tools
### Open-source and free access

- **Smina**  
  [![](https://img.shields.io/badge/source_forge-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://sourceforge.net/projects/smina)
- 🔥**AutoDock Suite**  
  [![](https://img.shields.io/badge/website-AB9FF2?style=flat&logo=temporal&labelColor=555555&logocolor=555555)](https://ccsb.scripps.edu/projects/docking)
  [![](https://img.shields.io/badge/AutoDock_Vina-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/ccsb-scripps/AutoDock-Vina/releases)
  ![Stars](https://img.shields.io/github/stars/ccsb-scripps/AutoDock-Vina?color=yellow&style=social)
- **AutoDock-GPU**  
  [![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/ccsb-scripps/AutoDock-GPU)
  ![Stars](https://img.shields.io/github/stars/ccsb-scripps/AutoDock-GPU?color=yellow&style=social)
- **MGLTools software suite**  
  [![](https://img.shields.io/badge/website-AB9FF2?style=flat&logo=temporal&labelColor=555555&logocolor=555555)](https://ccsb.scripps.edu/mgltools/)
  ![](https://img.shields.io/badge/free--access-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555) 
- **SwissDock**    
  [![](https://img.shields.io/badge/website-AB9FF2?style=flat&logo=temporal&labelColor=555555&logocolor=555555)](http://www.swissdock.ch/)
  ![](https://img.shields.io/badge/free--access-FD6F6F?style=flat&logo=darkreader&labelColor=555555&logocolor=555555)  
- **rDock**  
  [![](https://img.shields.io/badge/website-AB9FF2?style=flat&logo=temporal&labelColor=555555&logocolor=555555)](https://rdock.github.io)
  [![](https://img.shields.io/badge/code-38C26D?style=flat&logo=GitHub&labelColor=555555)](https://github.com/CBDD/rDock)
  ![Stars](https://img.shields.io/github/stars/CBDD/rDock?color=yellow&style=social)

### Commercial tool

- **GOLD Suite**  
  [![](https://img.shields.io/badge/website-AB9FF2?style=flat&logo=temporal&labelColor=555555&logocolor=555555)](https://www.ch.cam.ac.uk/computing/software/gold-suite)
- **GLIDE**  
  [![](https://img.shields.io/badge/website-AB9FF2?style=flat&logo=temporal&labelColor=555555&logocolor=555555)](https://www.schrodinger.com/products/glide)
- **MOE-Dock**  
  [![](https://img.shields.io/badge/website-AB9FF2?style=flat&logo=temporal&labelColor=555555&logocolor=555555)](https://www.computabio.com/molecular-docking-service.html)


# awesome-pretrain-molecule-protein-papers

- [awesome-pretrain-molecule-protein-papers](#awesome-pretrain-molecule-protein-papers)
- [Molecule](#molecule)
  - [Survey](#survey)
  - [Molecule Pretrain Model / Representation Learning](#molecule-pretrain-model--representation-learning)
  - [Molecule Generative Model](#molecule-generative-model)
  - [Molecule Protein Docking Model](#molecule-protein-docking-model)
  - [Tetrahedral Molecular Geometry](#tetrahedral-molecular-geometry)
  - [Chemical Reaction](#chemical-reaction)
  - [Drug Discovery](#drug-discovery)
  - [Chemistry Toolkits](#chemistry-toolkits)
  - [Molecule Dataset](#molecule-dataset)
  - [Materials](#materials)
  - [Molecular Dynamics](#molecular-dynamics)
- [Protein](#protein)
  - [Protein Pretrain Model / Representation Learning](#protein-pretrain-model--representation-learning)
  - [Protein Evolution](#protein-evolution)
  - [Protein Protein Docking Model](#protein-protein-docking-model)
  - [Protein Generative Model](#protein-generative-model)
  - [Protein Dataset](#protein-dataset)
- [3D Model](#3d-model)
  - [Equivariant Model](#equivariant-model)

# Molecule
## Survey
1. [2022 Current Opinion in Structural Biology] **Deep learning approaches for de novo drug design: An overview** [[paper]](https://doi.org/10.1016/j.sbi.2021.10.001)
2. [2022 arXiv] **Geometrically Equivariant Graph Neural Networks: A Survey** [[paper]](https://arxiv.org/pdf/2202.07230.pdf)
3. [2022 arXiv] **MolGenSurvey:A Systematic Survey in Machine Learning Models for Molecule Design** [[paper]](https://arxiv.org/abs/2203.14500)
4. [2023 Engineering] **Machine Learning for Chemistry: Basics and Applications** [[paper]](https://www.sciencedirect.com/science/article/pii/S2095809923002813)
5. [2023 IJCAI] **A Systematic Survey of Chemical Pre-trained Models** [[paper]](https://sxkdz.github.io/files/publications/IJCAI/CPM/CPM.pdf)[[code]](https://github.com/junxia97/awesome-pretrain-on-molecules)
6. [2023 Wiley] **Generative models for molecular discovery: Recent advances and challenges** [[paper]](https://wires.onlinelibrary.wiley.com/doi/full/10.1002/wcms.1608)
7. [2024 arXiv] **A Survey of Geometric Graph Neural Networks: Data Structures, Models and Applications** [[paper]](https://arxiv.org/pdf/2403.00485.pdf)
## Molecule Pretrain Model / Representation Learning

1. [2023 ICLR] **Uni-Mol: A Universal 3D Molecular Representation Learning Framework** [[paper]](https://openreview.net/forum?id=6K2RM6wVqKu)[[code]](https://github.com/dptech-corp/Uni-Mol.)
2. [2023 ICLR] **One Transformer Can Understand Both 2D & 3D Molecular Data** [[paper]](https://arxiv.org/pdf/2210.01765.pdf)[[code]](https://github.com/lsj2408/Transformer-M)
3. [2023 KDD] **Automated 3D Pre-Training for Molecular Property Prediction** [[paper]](https://arxiv.org/pdf/2306.07812.pdf)[[code]](https://github.com/LARS-research/3D-PGT)
4. [2022 KDD] **Unified 2D and 3D Pre-Training of Molecular Representations** [[paper]](https://arxiv.org/pdf/2207.08806.pdf)[[code]](https://github.com/teslacool/UnifiedMolPretrain)
5. [2023 KDD] **Dual-view Molecule Pre-training** [[paper]](https://dl.acm.org/doi/pdf/10.1145/3580305.3599317)[[code]](https://github.com/microsoft/DVMP)
6. [2022 arXiv] **CoSP: Co-supervised pretraining of pocket and ligand** [[paper]](https://arxiv.org/pdf/2206.12241.pdf)
7. [2022 Nature Machine Intelligence] **Molecular contrastive learning of representations via graph neural networks** [[paper]](https://arxiv.org/pdf/2102.10056.pdf)[[code]](https://github.com/yuyangw/MolCLR)
8. [2023 ICML] **Enhancing Activity Prediction Models in Drug Discovery with the Ability to Understand Human Language** [[paper]](https://openreview.net/forum?id=oeRMR0La70)
9. [Briefings in Bioinformatics] **MG-BERT: leveraging unsupervised atomic representation learning for molecular property prediction** [[paper]](https://www.researchgate.net/profile/Dong-Sheng-Cao/publication/351363304_MG-BERT_leveraging_unsupervised_atomic_representation_learning_for_molecular_property_prediction/links/609743a9458515d31507d9e2/MG-BERT-leveraging-unsupervised-atomic-representation-learning-for-molecular-property-prediction.pdf)[[code]](https://github.com/zhang-xuan1314/Molecular-graph-BERT)
10. [2021 NIPS] **Motif-based Graph Self-Supervised Learning for Molecular Property Prediction** [[paper]](https://arxiv.org/pdf/2110.00987.pdf)[[code]](https://github.com/zaixizhang/MGSSL)
11. [2022 ICLR] **Spherical Message Passing for 3D Molecular Graphs** [[paper]](https://openreview.net/pdf?id=givsRXsOt9r)[[code]](https://github.com/divelab/DIG)
12. [2022 NIPS] **ComENet: Towards Complete and Efficient Message Passing for 3D Molecular Graphs** [[paper]](https://openreview.net/pdf?id=mCzMqeWSFJ)[[code]](https://github.com/divelab/DIG)
13. [2022 Nvidia] **MegaMolBART** [[web-link]](https://docs.nvidia.com/launchpad/ai/clara-megamolbart/latest/clara-megamolbart-overview.html)[[code]](https://github.com/NVIDIA/MegaMolBART)
14. [2020 NIPS workshop] **Message Passing Networks for Molecules with Tetrahedral Chirality** [[paper]](https://arxiv.org/pdf/2012.00094.pdf)[[code]](https://github.com/PattanaikL/chiral_gnn)
15. [2023 arXiv] **Augmenting large language models with chemistry tools** [[paper]](https://arxiv.org/pdf/2304.05376.pdf)[[code]](https://github.com/ur-whitelab/chemcrow-public)
16. [2023 ICML workshop] **Extracting Molecular Properties from Natural Language with Multimodal Contrastive Learning** [[paper]](https://arxiv.org/pdf/2307.12996.pdf)

## Molecule Generative Model
1. [2021 NIPS] **Geomol: Torsional geometric generation of molecular 3d conformer ensembles** [[paper]](https://proceedings.neurips.cc/paper/2021/file/725215ed82ab6306919b485b81ff9615-Paper.pdf)[[code]](https://github.com/PattanaikL/GeoMol)
2. [2021 NIPS] **Predicting molecular conformation via dynamic graph score matching**
3. [2021 ICML] **GraphDF: A Discrete Flow Model for Molecular Graph Generation** [[paper]](https://proceedings.mlr.press/v139/luo21a.html)[[code]](https://github.com/divelab/DIG)
4. [2023 ICLR workshop] **Improving Small Molecule Generation using Mutual Information Machine** [[paper]](https://openreview.net/forum?id=iOJlwUTUyrN)
5. [2020 KDD] **MoFlow: An Invertible Flow Model for Generating Molecular Graphs** [[paper]](https://dl.acm.org/doi/pdf/10.1145/3394486.3403104)[[code]](https://github.com/calvin-zcx/moflow)
6. [2022 ICLR] **Geodiff: A geometric diffusion model for molecular conformation generation** [[paper]](https://arxiv.org/pdf/2203.02923.pdf)[[code]](https://github.com/MinkaiXu/GeoDiff)
7. [2022 ICML] **Equivariant Diffusion for Molecule Generation in 3D** [[paper]](https://arxiv.org/pdf/2203.17003.pdf)[[code]](https://github.com/ehoogeboom/e3_diffusion_for_molecules)
8. [2022 NIPS] **Torsional Diffusion for Molecular Conformer Generation** [[paper]](https://proceedings.neurips.cc/paper_files/paper/2022/file/994545b2308bbbbc97e3e687ea9e464f-Paper-Conference.pdf)[[code]](https://github.com/gcorso/torsional-diffusion)
9. [2022 ICLR] **An Autoregressive Flow Model for 3D Molecular Geometry Generation from Scratch** [[paper]](https://openreview.net/pdf?id=C03Ajc-NS5W)[[code]](https://github.com/divelab/DIG)
10. [2022 Research Square] **Accurate Protein-Ligand Complex Structure Prediction using Geometric Deep Learning** [[paper]](https://assets.researchsquare.com/files/rs-1454132/v1_covered.pdf?c=1647883536)[[code]](https://github.com/KeleiHe/LigPose)
11. [2022 arXiv] **BARTSmiles: Generative Masked Language Models for Molecular Representations** [[paper]](https://arxiv.org/abs/2211.16349)[[code]](https://github.com/YerevaNN/BARTSmiles/)
12. [2023 ICLR] **Diffdock: Diffusion steps, twists, and turns for molecular docking** [[paper]](https://arxiv.org/pdf/2210.01776.pdf%3C/p%3E)[[code]](https://github.com/gcorso/DiffDock)
13. [2023 ICLR] **De novo molecular generation via connection-aware motif mining** [[paper]](https://arxiv.org/pdf/2302.01129.pdf)[[code]](https://github.com/MIRALab-USTC/AI4Sci-MiCaM)
14. [2023 ICML] **MolDiff: Addressing the Atom-Bond Inconsistency Problem in 3D Molecule Diffusion Generation** [[paper]](https://arxiv.org/pdf/2305.07508.pdf)[[code]](https://github.com/pengxingang/MolDiff)
15. [2023 ICML] **Geometric Latent Diffusion Models for 3D Molecule Generation** [[paper]](https://arxiv.org/pdf/2305.01140.pdf)[[code]](https://github.com/MinkaiXu/GeoLDM)
16. [2023 ICLR] **Conditional antibody design as 3d equivariant graph translation** [[paper]](https://arxiv.org/pdf/2208.06073.pdf)[[code]](https://github.com/THUNLP-MT/MEAN)
17. [2023 bioRxiv] **Guided Diffusion for molecular generation with interaction prompt** [[paper]](https://www.biorxiv.org/content/10.1101/2023.09.11.557141v1.abstract)
18. [2023 Nature Machine Intelligence] **ResGen is a pocket-aware 3D molecular generation model based on parallel multiscale modelling** [[paper]](https://www.nature.com/articles/s42256-023-00712-7)[[code]](https://github.com/HaotianZhangAI4Science/ResGen)
19. [2023 Nature Computational Science] **Learning on topological surface and geometric structure for 3D molecular generation** [[paper]](https://www.nature.com/articles/s43588-023-00530-2)[[code]](https://github.com/HaotianZhangAI4Science/SurfGen)
20. [2023 arXiv] **Accurate transition state generation with an object-aware equivariant elementary reaction diffusion model** [[paper]](https://arxiv.org/pdf/2304.06174.pdf)
21. [2023 Nature Computational Science] **Guided diffusion for inverse molecular design** [[paper]](https://www.nature.com/articles/s43588-023-00532-0)[[code]](https://gitlab.com/porannegroup/gaudi)
22. [2023 JACS] **Generative Models as an Emerging Paradigm in the Chemical Sciences** [[paper]](https://pubs.acs.org/doi/10.1021/jacs.2c13467)
23. [2023 arXiv] **Generating Molecular Conformer Fields** [[paper]](https://arxiv.org/abs/2311.17932)
24. [2023 ICLR] **Retrieval-based Controllable Molecule Generation** [[paper]](https://arxiv.org/abs/2208.11126)[[code]](https://github.com/NVlabs/RetMol)
25. [2024 ICLR] **Unified Generative Modeling of 3D Molecules with Bayesian Flow Networks** [[paper]](https://openreview.net/forum?id=NSVtmmzeRB)
26. [2024 ICLR] **Training-free Multi-objective Diffusion Model for 3D Molecule Generation** [[paper]](https://openreview.net/forum?id=X41c4uB4k0)
27. [2024 Nature Machine Intelligence] **Generation of 3D molecules in pockets via a language model** [[paper]](https://github.com/stonewiseAIDrugDesign/Lingo3DMol)


## Molecule Protein Docking Model

1. [2015 Bioinformatics] **Fast, accurate, and reliable molecular docking with QuickVina 2** [[paper]](https://academic.oup.com/bioinformatics/article/31/13/2214/195750)
2. [2016 JCIM] **Protein-Ligand Scoring with Convolutional Neural Networks** [[paper]](https://arxiv.org/pdf/1612.02751.pdf)
3. [2017 Scientific Reports] **Protein-Ligand Blind Docking Using QuickVina-W With Inter-Process Spatio-Temporal Integration** [[paper]](https://www.nature.com/articles/s41598-017-15571-7)
4. [2018 Journal of Cheminformatics] **P2Rank: machine learning based tool for rapid and accurate prediction of ligand binding sites from protein structure** [[paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-018-0285-8)[[code]](http://github.com/rdk/p2rank)
5. [2018 IJCAI] **Interpretable drug target prediction using deep neural representation** [[paper]](https://www.ijcai.org/proceedings/2018/0468.pdf)
6. [2019 JCIM] **Are the Apo Proteins Suitable for the Rational Discovery of Allosteric Drugs?** [[paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00735)
7. [2020 Journal of Cheminformatics] **Edock: blind protein–ligand docking by replica-exchange monte carlo simulation** [[paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-020-00440-9)[[web-link]](https://seq2fun.dcmb.med.umich.edu//EDock/)
8. [2020 Journal of Cheminformatics] **spyrmsd: symmetry-corrected RMSD calculations in Python** [[paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-020-00455-2)[[code]](https://github.com/RMeli/spyrmsd)
9. [2021 Journal of Cheminformatics] **GNINA 1.0: molecular docking with deep learning** [[paper]](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-021-00522-2)[[code]](https://github.com/gnina/gnina)
10. [2021 Briefings in Bioinformatics] **InstaDock: A single-click graphical user interface for molecular docking-based virtual high-throughput screening** [[paper]](https://doi.org/10.1093/bib/bbaa279)[[web-link]](https://www.hassanlab.org/instadock)
11. [2021 Nature Machine Intelligence] **A geometric deep learning approach to predict binding conformations of bioactive molecules** [[paper]](https://www.nature.com/articles/s42256-021-00409-9)[[code]](https://github.com/OptiMaL-PSE-Lab/DeepDock)
12. [2022 ICML] **EQUIBIND: Geometric Deep Learning for Drug Binding Structure Prediction** [[paper]](https://proceedings.mlr.press/v162/stark22b/stark22b.pdf)[[code]](https://github.com/HannesStark/EquiBind)
13. [2022 NIPS] **TANKBind: Trigonometry-Aware Neural NetworKs for Drug-Protein Binding Structure Prediction** [[paper]](https://proceedings.neurips.cc/paper_files/paper/2022/file/2f89a23a19d1617e7fb16d4f7a049ce2-Paper-Conference.pdf)[[code]](https://github.com/luwei0917/TankBind)
14. [2022 Molecular Systems Biology] **Benchmarking AlphaFold-enabled molecular docking predictions for antibiotic discovery** [[paper]](https://www.embopress.org/doi/full/10.15252/msb.202211081)
15. [2022 arXiv] **CoSP: Co-supervised pretraining of pocket and ligand** [[paper]](https://arxiv.org/pdf/2206.12241.pdf)
16. [2023 ICLR] **Diffdock: Diffusion steps, twists, and turns for molecular docking** [[paper]](https://arxiv.org/pdf/2210.01776.pdf%3C/p%3E)[[code]](https://github.com/gcorso/DiffDock)
17. [2023 ICLR] **E3Bind: An End-to-End Equivariant Network for Protein-Ligand Docking** [[paper]](https://arxiv.org/pdf/2210.06069.pdf)
18. [2023 PNAS] **Contrastive learning in protein language space predicts interactions between drugs and protein targets** [[paper]](https://www.pnas.org/doi/abs/10.1073/pnas.2220778120)[[code]](https://github.com/samsledje/ConPLex_dev)
19. [2023 Nature Computer Science] **Efficient and accurate large library ligand docking with KarmaDock** [[paper]](https://www.nature.com/articles/s43588-023-00511-5)[[code]](https://github.com/schrojunzhang/KarmaDock)
20. [2023 JCIM] **BMaps: A Web Application for Fragment-Based Drug Design andCompound Binding Evaluation** [[paper]](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c00209)[[web-link]](https://www.boltzmannmaps.com/)
21. [2023 JCTC] **Equivariant Flexible Modeling of the Protein–Ligand Binding Pose with Geometric Deep Learning** [[paper]](https://pubs.acs.org/doi/epdf/10.1021/acs.jctc.3c00273)[[code]](https://github.com/tiejundong/FlexPose)
22. [2023 arXiv] **Generalist Equivariant Transformer Towards 3D Molecular Interaction Learning** [[paper]](https://arxiv.org/pdf/2306.01474.pdf)
23. [2023 Openreview] **The Discovery of Binding Modes Requires Rethinking Docking Generalization** [[paper]](https://openreview.net/pdf?id=UfBIxpTK10)
24. [2023 Openreview] **Protein-Ligand Interaction Prior for Binding-aware 3D Molecule Diffusion Models** [[paper]](https://openreview.net/forum?id=qH9nrMNTIW)
25. [2023 arXiv] **Generating Molecular Fragmentation Graphs with Autoregressive Neural Networks** [[paper]](https://arxiv.org/pdf/2304.13136.pdf)[[code]](https://github.com/samgoldman97/ms-pred)

## Tetrahedral Molecular Geometry

1. [2020 NIPS workshop] **Message Passing Networks for Molecules with Tetrahedral Chirality** [[paper]](https://arxiv.org/pdf/2012.00094.pdf)[[code]](https://github.com/PattanaikL/chiral_gnn)
2. [2023 Nature Reviews Chemistry] **Detection and analysis of chiral molecules as disease biomarkers** [[paper]](https://www.nature.com/articles/s41570-023-00476-z)

## Chemical Reaction
1. [2018 Nature] **Planning chemical syntheses with deep neural networks and symbolic AI** [[paper]](https://www.nature.com/articles/nature25978)[[code-nonofficial]](https://github.com/linminhtoo/neuralsym)
2. [2020 JCIM] **Predicting Retrosynthetic Reaction using Self-Corrected Transformer Neural Networks** [[paper]](https://pubs.acs.org/doi/full/10.1021/acs.jcim.9b00949)[[code]](https://github.com/Jh-SYSU/SCROP)
3. [2020 Chemical Science] **Automatic retrosynthetic route planning using template-free models** [[paper]](https://pubs.rsc.org/en/content/articlehtml/2020/sc/c9sc03666k)[[code]](https://github.com/PKUMDL-AI/AutoSynRoute)
4. [2023 Nature Communications] **Retrosynthesis prediction with an interpretable deep-learning framework based on molecular assembly tasks** [[paper]](https://www.nature.com/articles/s41467-023-41698-5)[[code]](https://github.com/wangyu-sd/RetroExplainer)
5. [2023 Nature Machine Intelligence] **Bridging the Gap between Chemical Reaction Pretraining and Conditional Molecule Generation with a Unified Model** [[paper]](https://www.nature.com/articles/s42256-023-00764-9)[[code]](https://github.com/qiangbo1222/Uni-RXN-official)
6. [2024-ICLR] **RetroBridge: Modeling Retrosynthesis with Markov Bridges** [[paper]](https://arxiv.org/abs/2308.16212)

## Drug Discovery
1. [2021 JCIM] **OpenChem: A Deep Learning Toolkit for Computational Chemistry and Drug Design** [[paper]](https://pubs.acs.org/doi/abs/10.1021/acs.jcim.0c00971)
1. [2023 Nature Communications] **Discovery of senolytics using machine learning** [[paper]](https://www.nature.com/articles/s41467-023-39120-1)[[code]](https://zenodo.org/record/7870357)
2. [2023 ICML] **Drug Discovery under Covariate Shift with Domain-Informed Prior Distributions over Functions** [[paper]](https://openreview.net/pdf?id=BbZVFj0QPv)[[code]](https://github.com/leojklarner/Q-SAVI)
3. [2023 Current Opinion in Structural Biology] **Structure-based drug design with geometric deep learning** [[paper]](https://www.sciencedirect.com/science/article/pii/S0959440X23000222)
4. [2023 Openreview] **Drug Discovery with Dynamic Goal-aware Fragments** [[paper]](https://openreview.net/forum?id=sLGliHckR8)
5. [2023 ICML] **DecompDiff: Diffusion Models with Decomposed Priors for Structure-Based Drug Design** [[paper]](https://openreview.net/forum?id=9qy9DizMlr)[[code]](https://github.com/bytedance/DecompDiff)
6. [2023 Openreview] **Streamlining Generative Models for Structure-Based Drug Design** [[paper]](https://openreview.net/forum?id=8DLVrWL78S)
7. [2023 Openreview] **Long-Short-Range Message-Passing: A Fragmentation-Based Framework to Capture Non-Local Atomistic Interactions** [[paper]](https://openreview.net/forum?id=rvDQtdMnOl)
8. [2023 ACS Central Science] **Geometric Deep Learning for Structure-Based Ligand Design** [[paper]](https://doi.org/10.1021/acscentsci.3c00572)
9. [2023 ICLR] **De Novo Molecular Generation via Connection-aware Motif Mining** [[paper]](https://arxiv.org/abs/2302.01129)[[code]](https://github.com/MIRALab-USTC/AI4Sci-MiCaM)
10. [2023 NIPS] **Tartarus: A benchmarking platform for realistic and practical inverse molecular design** [[paper]](https://arxiv.org/abs/2209.12487) [[code]](https://github.com/aspuru-guzik-group/Tartarus)
11. [2023 arXiv] **ChatGPT-powered Conversational Drug Editing Using Retrieval and Domain Feedback** [[paper]](https://arxiv.org/pdf/2305.18090.pdf)[[code]](https://github.com/chao1224/ChatDrug)
12. [2023 ACS Central Science] **2023-ACS Central Science / Geometric Deep Learning for Structure-Based Ligand Design** [[paper]](https://pubs.acs.org/doi/full/10.1021/acscentsci.3c00572)
13. [2023 Wiley] **Generative models for molecular discovery: Recent advances and challenges** [[paper]](https://wires.onlinelibrary.wiley.com/doi/full/10.1002/wcms.1608)

## Chemistry Toolkits

1. [RDKit](http://www.rdkit.org/): **Open-source cheminformatics software**
2. [PyMOL](https://pymol.org/2/): **A user-sponsored molecular visualization system on an open-source foundation**
3. [CDK](https://cdk.github.io/): **Chemistry Development Kit (Open Source modular Java libraries for Cheminformatics)**
4. [Open Babel](http://openbabel.org/wiki/Main_Page): **The Open Source Chemistry Toolbox**
5. [Cinfony](https://cinfony.github.io/): **A common API to several cheminformatics toolkits**
6. [Indigo](https://lifescience.opensource.epam.com/indigo/index.html): **A universal molecular toolkit that can be used for molecular fingerprinting, substructure search, and molecular visualization**
7. [ChemoPy](https://github.com/ifyoungnet/Chemopy): **A freely available python package for computational biology and chemoinformatics**
8. [ChemmineR](https://www.bioconductor.org/packages/release/bioc/html/ChemmineR.html): **A cheminformatics package for analyzing drug-like small molecule data in R**
9. [ChemKit](https://github.com/kylelutz/chemkit): **An open source software library for chemistry**
10. [DeepChem](https://deepchem.io/): **A Python library for machine learning and deep learning on molecular and quantum datasets**
11. [MolVS](https://molvs.readthedocs.io/en/latest/): **Molecular Validation and Standardization**
12. [spyrmsd](https://spyrmsd.readthedocs.io/en/develop/): **spyrmsd: symmetry-corrected RMSD calculations in Python**
13. [torchdrug](https://torchdrug.ai/): **A machine learning platform designed for drug discovery**
14. [torchprotein](https://torchprotein.ai/): **A machine learning library for protein science, built on top of TorchDrug**
15. [OpenChem](https://deepchem.io/): **A deep learning toolkit for computational chemistry and drug design**
## Molecule Dataset

1. [2012 JCIM] **Enumeration of 166 billion organic small molecules in the chemical universe database GDB-17** [[paper]](https://pubs.acs.org/doi/10.1021/ci300415d)
2. [2015 JCIM] **ZINC 15 – Ligand Discovery for Everyone** [[paper]](https://pubs.acs.org/doi/full/10.1021/acs.jcim.5b00559)
3. [2016 Nucleic Acids Research] **PubChem Substance and Compound databases**
4. [2017 Science Advances] **Machine learning of accurate energy-conserving molecular force fields** [[paper]](https://www.science.org/doi/10.1126/sciadv.1603015)[[data-link]](http://quantum-machine.org/datasets/)
5. [2018 Nucleic Acids Research] **ChEMBL: towards direct deposition of bioassay data** [[paper]](https://academic.oup.com/nar/article/47/D1/D930/5162468)
6. [2018 Nucleic Acids Research] **DrugBank 5.0: a major update to the DrugBank database for 2018** [[paper]](https://academic.oup.com/nar/article/46/D1/D1074/4602867)
7. [2018 JCIM] **Comparative Assessment of Scoring Functions: The CASF-2016 Update** [[paper]](https://pubs.acs.org/doi/abs/10.1021/acs.jcim.8b00545)[[web-link]](http://www.pdbbind.org.cn/casf.php)
8. [2019 JCIM] **GuacaMol: Benchmarking Models for De Novo Molecular Design** [[paper]](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00839)[[code]](https://github.com/BenevolentAI/guacamol)[[web-link]](https://benevolent.ai/guacamol)
9. [2023 arXiv] **Towards Foundational Models for Molecular Learning on Large-Scale Multi-Task Datasets** [[paper]](https://arxiv.org/pdf/2310.04292v2.pdf)


|                                                 Dataset Name                                                 |  #Molecules  |                   Property                   |                       About Data                       |
| :-----------------------------------------------------------------------------------------------------------: | :-----------------: | :-------------------------------------------: | :-----------------------------------------------------: |
|                                    [GDB-17](https://gdb.unibe.ch/downloads/)                                    |        166B        |           small organical molecules           |               chemical universe database               |
| [QM9](https://figshare.com/collections/Quantum_chemistry_structures_and_properties_of_134_kilo_molecules/978904) |       133,885       |          quantum chemical properties          |                   a subset of GDB-17                   |
|                                      [ZINC15](https://zinc15.docking.org/)                                      | 230M(3D), over 750M |        bioactivity of small molecules        |          biologically relevant small molecules          |
|                               [GuacaMol](https://github.com/BenevolentAI/guacamol)                               |         N/A         |   benchmark suite for generative chemistry   |     a subset of molecules extracted from ChEMBL 24     |
|                                     [ChEMBL](https://www.ebi.ac.uk/chembl/)                                     |        2.4M        | bioactive molecules with drug-like properties |     contains chemical, bioactivity and genomic data     |
|                              [PubChem BioAssay](https://pubchem.ncbi.nlm.nih.gov/)                              |     270,998,024     |               PubChem BioAssay               |                   a subset of PubChem                   |
|                              [PubChem Compound](https://pubchem.ncbi.nlm.nih.gov/)                              |      1,366,263      |               PubChem Compound               |                   a subset of PubChem                   |
|                              [PubChem Substance](https://pubchem.ncbi.nlm.nih.gov/)                              |     109,891,994     |               PubChem Substance               |                   a subset of PubChem                   |
|                                     [DrugBank 5.0](https://go.drugbank.com/)                                     |         N/A         |                   drug data                   |       bioinformatics and cheminformatics database       |
|                                 [GEOM-Drugs](https://doi.org/10.7910/DVN/JNGTDF)                                 |      430,00 0      |                   drug data                   | a larger scale dataset of molecular conformers than QM9 |
<!-- |CrossDocked2020| -->
<!-- |[PDBBind](http://www.pdbbind.org.cn/index.php?action=showall)| 23,496 biomolecular complexes | binding affinity data (protein-protein, protein-ligand, protein-acid)| a subset of PDB database | 
| [MD17](https://archive.materialscloud.org/record/2020.82)| ||| -->

## Materials
1. [2023 Openreview] **Crystalformer: Infinitely Connected Attention for Periodic Structure Encoding** [[paper]](https://openreview.net/forum?id=fxQiecl9HB)
2. [2023 Openreview] **Scalable Diffusion for Materials Generation** [[paper]](https://openreview.net/forum?id=wm4WlHoXpC)
3. [2023 Openreview] **MOFDiff: Coarse-grained Diffusion for Metal-Organic Framework Design** [[paper]](https://openreview.net/forum?id=0VBsoluxR2)
4. [2023 JACS] **MOFormer: Self-Supervised Transformer Model for Metal–Organic Framework Property Prediction** [[paper]](https://pubs.acs.org/doi/full/10.1021/jacs.2c11420)[[code]](https://github.com/zcao0420/MOFormer)

## Molecular Dynamics
1. [2023 arXiv] **Score dynamics: scaling molecular dynamics with picoseconds timestep via conditional diffusion model** [[paper]](https://arxiv.org/pdf/2310.01678.pdf)[[code]](https://github.com/llnl/graphite)
# Protein

## Protein Pretrain Model / Representation Learning

1. [2020 bioRxiv] **Transformer protein language models are unsupervised structure learners** [[paper]](https://www.biorxiv.org/content/10.1101/2020.12.15.422761v1.full.pdf)[[code]](https://github.com/facebookresearch/esm)
2. [2021 PNAS] **Biological Structure and Function Emerge from Scaling Unsupervised Learning to 250 Million Protein Sequences** [[paper]](https://www.pnas.org/doi/abs/10.1073/pnas.2016239118)[[code]](https://github.com/facebookresearch/esm)
3. [2021 ICML] **MSA Transformer** [[paper]](http://proceedings.mlr.press/v139/rao21a/rao21a.pdf)[[code]](https://github.com/facebookresearch/esm)
4. [2021 NIPS] **Language models enable zero-shot prediction of the effects of mutations on protein function** [[paper]](https://proceedings.neurips.cc/paper_files/paper/2021/file/f51338d736f95dd42427296047067694-Paper.pdf)[[code]](https://github.com/facebookresearch/esm)
5. [2021 Nature] **Highly accurate protein structure prediction with AlphaFold** [[paper]](https://www.nature.com/articles/s41586-021-03819-2)[[code]](https://github.com/deepmind/alphafold)
6. [2021 Science] **Accurate prediction of protein structures and interactions using a three-track neural network** [[paper]](https://www.science.org/doi/10.1126/science.abj8754)[[code]](https://github.com/RosettaCommons/RoseTTAFold)
7. [2022 ICML] **Learning inverse folding from millions of predicted structures** [[paper]](https://proceedings.mlr.press/v162/hsu22a/hsu22a.pdf)[[code]](https://github.com/deepmind/alphafold)
8. [2022 bioRxiv] **OpenFold: Retraining AlphaFold2 yields new insights into its learning mechanisms and capacity for generalization** [[paper]](https://www.biorxiv.org/content/10.1101/2022.11.20.517210v3.full.pdf)[[code]](https://github.com/aqlaboratory/openfold)
9. [2021 PNAS] **Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences** [[paper]](https://www.pnas.org/doi/abs/10.1073/pnas.2016239118)[[code]](https://github.com/facebookresearch/esm)
10. [2022 bioRxiv] **Language models of protein sequences at the scale of evolution enable accurate structure prediction** [[paper]](https://www.biorxiv.org/content/10.1101/2022.07.20.500902v1.full.pdf?utm_campaign=M2D2%20Community%20Round-Up&utm_medium=email&utm_source=Revue%20newsletter)
11. [2023 Science] **Evolutionary-scale prediction of atomic-level protein structure with a language model** [[paper]](https://www.science.org/doi/abs/10.1126/science.ade2574)[[code]](https://github.com/facebookresearch/esm)
12. [2022 Nature Communications] **ProtGPT2 is a deep unsupervised language model for protein design** [[paper]](https://www.nature.com/articles/s41467-022-32007-7)[[code]](https://huggingface.co/docs/transformers/main_classes/trainer)
13. [2023 bioRxiv] **xTrimoPGLM: Unified 100B-Scale Pre-trained Transformer for Deciphering the Language of Protein** [[paper]](https://www.biorxiv.org/content/10.1101/2023.07.05.547496v3)
14. [2023 arXiv] **Generative Pretrained Autoregressive Transformer Graph Neural Network applied to the Analysis and Discovery of Novel Proteins** [[paper]](https://arxiv.org/pdf/2305.04934.pdf)[[code]](https://github.com/lamm-mit/MateriomicTransformer)
15. [2023 bioRxiv] **Pretrainable Geometric Graph Neural Network for Antibody Affinity Maturation** [[paper]](https://www.biorxiv.org/content/10.1101/2023.08.10.552845v1.full.pdf)
16. [2023 Openreview] **Pre-training Sequence, Structure, and Surface Features for Comprehensive Protein Representation Learning** [[paper]](https://openreview.net/forum?id=BEH4mGo7zP)
17. [2023 ICLR] **Protein Representation Learning by Geometric Structure Pretraining** [[paper]](https://arxiv.org/abs/2203.06125)[[code]](https://github.com/DeepGraphLearning/GearNet)

## Protein Evolution 
1. [2023 Nature] **Clustering-predicted structures at the scale of the known protein universe** [[paper]](https://www.nature.com/articles/s41586-023-06510-w)

## Protein Protein Docking Model
1. [2020 Sturcture] **Performance and its limits in rigid body protein-protein docking** [[paper]](https://www.cell.com/structure/pdf/S0969-2126(20)30209-4.pdf)[[web-link]](https://cluspro.bu.edu/login.php)
2. [2020 Nature Protocols] **The HDOCK server for integrated protein–protein docking** [[paper]](https://www.nature.com/articles/s41596-020-0312-x)[[web-link]](http://hdock.phys.hust.edu.cn/)
3. [2021 bioRxiv] **Improved docking of protein models by a combination of alphafold2 and cluspro** [[paper]](https://www.biorxiv.org/content/10.1101/2021.09.07.459290v2.full.pdf)
4. [2022 ICLR] **Independent SE(3)-Equivariant Models for End-to-End Rigid Protein Docking** [[paper]](https://arxiv.org/pdf/2111.07786.pdf)[[code]](https://github.com/octavian-ganea/equidock_public)
## Protein Generative Model

1. [2023 ICLR] **Learning Hierarchical Protein Representations via Complete 3D Graph Networks** [[paper]](https://openreview.net/forum?id=9X-hgLDLYkQ)[[code]](https://github.com/divelab/DIG)
2. [2023 ICML] **Se(3) diffusion model with application to protein backbone generation** [[paper]](https://arxiv.org/pdf/2302.02277.pdf)[[code]](https://github.com/jasonkyuyim/se3_diffusion)
3. [2023 ICLR] **Diffusion probabilistic modeling of protein backbones in 3d for the motif- scaffolding problem** [[paper]](https://openreview.net/forum?id=6TxBxqNME1Y)[[code]](https://github.com/blt2114/ProtDiff_SMCDiff)
4. [2023 ICLR] **Protein sequence and structure co-design with equivariant translation** [[paper]](https://arxiv.org/pdf/2210.08761.pdf)
5. [2023 ICML] **SE(3) diffusion model with application to protein backbone generation** [[paper]](https://arxiv.org/pdf/2302.02277.pdf)[[code]](https://github.com/jasonkyuyim/se3_diffusion)
6. [2023 arXiv] **DiffPack: A Torsional Diffusion Model for Autoregressive Protein Side-Chain Packing** [[paper]](https://arxiv.org/pdf/2306.01794.pdf)
7. [2023 bioRxiv] **Protein generation with evolutionary diffusion: sequence is all you need** [[paper]](https://www.biorxiv.org/content/10.1101/2023.09.11.556673v1.abstract)
8. [2023 Openreview] **SE(3)-Stochastic Flow Matching for Protein Backbone Generation** [[paper]](https://openreview.net/forum?id=kJFIH23hXb)
9. [2023 Nature] **De novo design of protein structure and function with RFdiffusion** [[paper]](https://www.nature.com/articles/s41586-023-06415-8)[[code]](https://github.com/RosettaCommons/RFdiffusion)

## Protein Dataset

# 3D Model

## Equivariant Model

1. [2013 Doc] **Lie groups for 2d and 3d transformations** [[paper]](https://ethaneade.org/lie.pdf)
2. [2018 arXiv] **Tensor field networks: Rotation- and translation-equivariant neural networks for 3D point clouds** [[paper]](https://arxiv.org/pdf/1802.08219.pdf)[[code]](https://github.com/tensorfieldnetworks/tensorfieldnetworks)
3. [2019 NIPS] **Cormorant: Covariant Molecular Neural Networks** [[paper]](https://arxiv.org/abs/1906.04015)[[code]](https://github.com/risilab/cormorant)
4. [2020 NIPS] **SE(3)-Transformers: 3D Roto-Translation Equivariant Attention Networks** [[paper]](https://arxiv.org/abs/2006.10503)[[code]](https://github.com/FabianFuchsML/se3-transformer-public)
5. [2021 ICML] **E(n) Equivariant Graph Neural Networks** [[paper]](http://proceedings.mlr.press/v139/satorras21a/satorras21a.pdf)[[code]](https://github.com/vgsatorras/egnn)
6. [2021 Nature Communications] **E(3)-equivariant graph neural networks for data-efficient and accurate interatomic potentials** [[paper]](https://www.nature.com/articles/s41467-022-29939-5)[[code]](https://github.com/mir-group/nequip)
7. [2022 ICLR] **Geometric and Physical Quantities Improve E(3) Equivariant Message Passing** [[paper]](https://arxiv.org/abs/2110.02905)
8. [2022 ICLR workshop] **Denoising diffusion probabilistic models on so (3) for rotational alignment** [[paper]](https://openreview.net/pdf?id=BY88eBbkpe5)
9. [2022 arXiv] **e3nn: Euclidean Neural Networks** [[paper]](https://arxiv.org/pdf/2207.09453.pdf)
10. [2023 ICLR] **Equiformer: Equivariant Graph Attention Transformer for 3D Atomistic Graphs** [[paper]](https://arxiv.org/pdf/2206.11990.pdf)[[code]](https://github.com/atomicarchitects/equiformer)
11. [2024 ICLR] **Hybrid Directional Graph Neural Network for Molecules** [[paper]](https://openreview.net/forum?id=BBD6KXIGJL)[[code]](https://github.com/AnonymousACode/HDGNN)
